install_config/install/quick_install.adoc

```diff
@@ -142,44 +142,54 @@ xref:running-an-unattended-installation[unattended installation].
 ====
 [source,yaml]
 ----
-version: v1 <1>
+version: v2 <1>
 variant: openshift-enterprise <2>
 variant_version: 3.2 <3>
-ansible_ssh_user: root <4>
-ansible_log_path: /tmp/ansible.log <5>
-hosts: <6>
-- ip: 10.0.0.1 <7>
-  hostname: master-private.example.com <7>
-  public_ip: 24.222.0.1 <8>
-  public_hostname: master.example.com <8>
-  master: true <9>
-  node: true <9>
-  containerized: true <10>
-  connect_to: 24.222.0.1 <11>
-- ip: 10.0.0.2
-  hostname: node1-private.example.com
-  public_ip: 24.222.0.2
-  public_hostname: node1.example.com
-  node: true
-  connect_to: 10.0.0.2
-- ip: 10.0.0.3
-  hostname: node2-private.example.com
-  public_ip: 24.222.0.3
-  public_hostname: node2.example.com
-  node: true
-  connect_to: 10.0.0.3
+ansible_log_path: /tmp/ansible.log <4>
+deployment:
+  ansible_ssh_user: root <5>
+  hosts: <6>
+  - ip: 10.0.0.1 <7>
+    hostname: master-private.example.com <7>
+    public_ip: 24.222.0.1 <8>
+    public_hostname: master.example.com <8>
+    roles: <9>
+      - master
+      - node
+    containerized: true <10>
+    connect_to: 24.222.0.1 <11>
+  - ip: 10.0.0.2
+    hostname: node1-private.example.com
+    public_ip: 24.222.0.2
+    public_hostname: node1.example.com
+    node_labels: {'region': 'infra'} <12>
+    roles:
+      - node
+    connect_to: 10.0.0.2
+  - ip: 10.0.0.3
+    hostname: node2-private.example.com
+    public_ip: 24.222.0.3
+    public_hostname: node2.example.com
+    roles:
+      - node
+    connect_to: 10.0.0.3
+  roles: <13>
+    master:
+      variable: "value" <14>
```

Comment:
Maybe to show the form of "value":

```
  roles: <13>
    master:
      variable: "<variable_name1>=<value1>" <14>
    node:
      variable: "<variable_name2>=<value2>, <variable_name3>=<value3>" <14>
```

^ Does it take a comma-separated list?